meteor/server/api/deviceTriggers/StudioDeviceTriggerManager.ts

```diff
@@ -224,6 +238,18 @@ export class StudioDeviceTriggerManager {
 		actionManager.deleteActionsOtherThan(touchedActionIds)
 	}
 
+	protected async updateTriggersFromLastCache(): Promise<void> {
```

Comment:
Why is this method `protected`? I don't see this being subclassed anywhere?

Also, I feel that maybe if there's a need to have `cache` be persisted, it makes more sense to just change the signature of the `updateTriggers` altogether and add a `setCache` method? I think I'll become confused next time I'll be looking at this code, trying to figure out which of these (`updateTriggersFromLastCache` or `updateTriggers`) I should use. I find that this PR breaks the state-less nature of `StudioDeviceTriggerManager` - which is fine, but if that's the case, let's just go all in.